Cilsil/Cil/Parsers/ExceptionParser.cs

```diff
@@ -0,0 +1,293 @@
+using Cilsil.Utils;
+using Cilsil.Sil;
+using Cilsil.Sil.Types;
+using Mono.Cecil.Cil;
+using Mono.Cecil;
+using Cilsil.Sil.Instructions;
+using Cilsil.Sil.Expressions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Cilsil.Cil.Parsers
+{
+    internal class ExceptionParser : InstructionParser
+    {
+        protected override bool ParseCilInstructionInternal(Instruction instruction,
+                                                            ProgramState state)
+        {
+            var endBlockOffset = state.ExceptionBlockStartToEndOffsets[state.CurrentInstruction.Offset];
+            
+            /*node1 preds: succs:2 3 exn: Instructions
+            n$47=*&amp;return:void [line 55];   # load return
+            *&amp;return:void=null [line 55];
```

Comment:
add comment talking about how, in case an exception is encountered, we translate the return output to being a null/none type

Reply:
Done